storage/statedb/cache.go

```diff
@@ -1,8 +1,77 @@
 package statedb
 
+import "errors"
+
+type CacheType string
+
+const (
+	// Available stateDB cache types
+	LocalCache  CacheType = "LocalCache"
+	RemoteCache           = "RemoteCache"
+	HybridCache           = "HybridCache"
+)
+
+var (
+	errNotSupportedCacheType = errors.New("not supported stateDB Cache type")
+)
+
 // Cache interface the cache of stateDB
 type Cache interface {
 	Set(k, v []byte)
 	Get(k []byte) []byte
 	Has(k []byte) ([]byte, bool)
 }
+
+// NewCache creates one type of any supported stateDB caches.
+// TODO-Klaytn: refine input parameters after setting node flags
+func NewCache(cacheType CacheType, maxBytes int, redisEndpoint []string, redisCluster bool) (Cache, error) {
```

Comment:
I think we don't need to define this `NewCache()` and cache type declares.
We can use `NewFastCache`, `NewRedisCache` and `NewHybridCache` directly.

Reply:
`cacheType` (or whatever selecting cache type) will be set by a user. Therefore, we need a selecting logic like `NewCache` in somewhere selecting one of `NewFastCache`, `NewRedisCache` and `NewHybridCache`.